### ISSUE TITLE
Wrong "typings" key in package.json after monorepo refactor.

### ISSUE THREAD

**Issue by adamduren** [firebase/firebase-admin-node#136]
## Reproduction steps
See [repo](https://github.com/adamduren/firebase-typings-issue) to reproduce.

## Description

The path for typings is missing the `cjs` directory.

```
"typings": "dist/index.d.ts",
```

Should be 
```
"typings": "dist/cjs/index.d.ts",
```

Noticed in the following places:

 * https://github.com/firebase/firebase-js-sdk/blob/master/packages/app/package.json#L50
 * https://github.com/firebase/firebase-js-sdk/blob/master/packages/messaging/package.json#L48
 * https://github.com/firebase/firebase-js-sdk/blob/master/packages/storage/package.json#L45

Results in the following error during compilation with `tsc`

```
node_modules/@firebase/database/dist/cjs/index.d.ts(16,35): error TS7016: Could not find a declaration file for module '@firebase/app'. '/workspace/functions/node_modules/@firebase/app/dist/cjs/index.js' implicitly has an 'any' type.
node_modules/@firebase/database/dist/cjs/src/api/Database.d.ts(3,46): error TS7016: Could not find a declaration file for module '@firebase/app'. '/workspace/functions/node_modules/@firebase/app/dist/cjs/index.js' implicitly has an 'any' type.
  Try `npm install @types/@firebase/app` if it exists or add a new declaration (.d.ts) file containing `declare module '@firebase/app';`
node_modules/@firebase/database/dist/cjs/src/core/AuthTokenProvider.d.ts(16,52): error TS7016: Could not find a declaration file for module '@firebase/app'.'/workspace/functions/node_modules/@firebase/app/dist/cjs/index.js' implicitly has an 'any' type.
  Try `npm install @types/@firebase/app` if it exists or add a new declaration (.d.ts) file containing `declare module '@firebase/app';`
node_modules/@firebase/database/dist/cjs/src/core/Repo.d.ts(3,29): error TS7016: Could not find a declaration file for module '@firebase/app'. '/workspace/functions/node_modules/@firebase/app/dist/cjs/index.js' implicitly has an 'any' type.
  Try `npm install @types/@firebase/app` if it exists or add a new declaration (.d.ts) file containing `declare module '@firebase/app';`
```

## Environment
 * Node Version: v8.9.1 (also tried 6.x, 7.x)
 * NPM Version: v5.5.1
 * TypeScript Version: v2.4.2 / v2.6.1

**Comment by adamduren** [comment]
Moved to `firebase-admin-node` from https://github.com/firebase/firebase-js-sdk/issues/318.

**Comment by 0x80** [comment]
@adamduren How do you work around this issue?

I've tried adding them to my tsconfig includes, but that doesn't seem to cut it.

```json
"include": [
    "src/**/*",

    // These paths are currently misconfigured in the firebase sdk
    "node_modules/@firebase/app/dist/cjs/index.d.ts",
    "node_modules/@firebase/util/dist/cjs/index.d.ts"
  ],
```

**Comment by 0x80** [comment]
Ah I get it now I think. 

skipLibCheck only works if you don't explicitly add those *.d.ts files to you own files list, because then they're not considered external lib anymore. Using the skipLibCheck without listing the files at least lets me compile my own code. 

But of course this is still a problem, because now all checks for all external modules are disabled.

**Comment by raDiesle** [comment]
already solved? It's closed, but still I get this issue and I do not see referenced code-fix.

**Comment by adamduren** [comment]
Issue appears open for me

**Comment by hiranya911** [comment]
This should be fixed in the 5.5.1 release. We no longer use the typings from `@firebase/database`.

@jshcrowthe is working separately on getting the typings of `@firebase/database` sorted out. Hopefully we will be able to consume them in the admin SDK in the future.
